typings/react-datetime-tests.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
```

Comment:
So in the DefinitelyTyped readme it says

> **File**: foo-tests.ts
> **Purpose**: This contains sample code which tests the typings. This code does not run, but it is type-checked.

And that's what's happening in your changes, right? I didn't get that at first but yeah if strictly the typings are being tested I can see them living in the folder they're in now. 

I see the typings being a separate modular thing on top of the component, so if this is the case (please confirm) then I'd say just leave it as is.